Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Stage 1: Builder Stage
```

Comment:
I think we need to explain why this exists. Something like "normally you don't need to run this in a docker, in fact when you use IDE plugins you get a binary file that just runs on the host without a container, but you might want to run this in a container for any reason that makes sense to you (such as isolation), and here's a Dockerfile to help you".